installers-conda/resources/installers-conda.patch

```diff
@@ -1,313 +0,0 @@
-From 40db10418cf9b46a70b48026d202d4474d62b99a Mon Sep 17 00:00:00 2001
```

Comment:
Why is this patch no longer necessary? I thought you needed to do several adjustments to our source code for the installers to work as expected.

Reply:
The patch is still necessary. However, I think it will be simpler to create the file on-demand rather than track and update it.

Currently, necessary changes are made on the `installers-conda-patch` branch, but then a separate PR must be made on the `5.x` branch to update the patch file when the `installers-conda-patch` gets new commits. With this PR we don't need to track and update a patch file on the `5.x` branch, rather the patch file is created when the spyder conda package is built.

Reply:
Ok, thanks for the clarification. I didn't understand where the patch went.